ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/JsonRpcMethodsFactory.java

```diff
@@ -107,8 +107,7 @@ public Map<String, JsonRpcMethod> methods(
                   blockchainQueries, protocolSchedule, transactionPool, privacyParameters),
               new ExecutionEngineJsonRpcMethods(
                   miningCoordinator,
-                  MergeProtocolSchedule.createTimestamp(
-                      genesisConfigOptions, privacyParameters, false),
+                  MergeProtocolSchedule.create(genesisConfigOptions, privacyParameters, false),
```

Comment:
Now that the MergeProtocolShedule already contains the timestamp forks this creation might not be needed anymore as discussed. Might be useful to try use the existing one.

Reply:
Created a task on the other PR https://github.com/hyperledger/besu/issues/5354